aws/request/handlers.go

```diff
@@ -20,22 +20,6 @@ type Handlers struct {
 	AfterRetry       HandlerList
 }
 
-// Copy returns of this handler's lists.
-func (h *Handlers) Copy() Handlers {
```

Comment:
Correct, the only time Copy will be called is when a service client is created from a `session.Session`, and then each time an API operation is called. Outside of that it should be very rare of a handler to be copied.

An alternative approach would be to institute lazy copy of the handlers, but this would require mutexes to wrap the HandlerList functions, since concurrent mutations need to be supported. I'm guessing adding the critical section check will have more overhead over the life time of the request than the copy.